app/models/moderator.ts

```diff
@@ -0,0 +1,26 @@
+import DS from 'ember-data';
+
+import OsfModel from './osf-model';
+import ProviderModel from './provider';
+
+const { attr, belongsTo } = DS;
+
+export enum PermissionGroup {
+    Admin = 'admin',
+    Moderator = 'moderator',
+}
+
+export default class ModeratorModel extends OsfModel {
+    @attr('string') permissionGroup!: PermissionGroup;
+    @attr('string') fullName!: string;
+    @attr('string') email!: string;
+
+    @belongsTo('moderator')
```

Comment:
Should belong to `provider` instead of `moderator`. Also, would this be a polymorphic `belongsTo` if it belongs to `provider` instead of `registration-provider`?